SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -3531,6 +3540,64 @@ bool USpatialNetDriver::HasTimedOut(const float Interval, uint64& TimeStamp)
 	return false;
 }
 
+void USpatialNetDriver::ServerReplicateActors_BuildConsiderList(TArray<FNetworkObjectInfo*>& OutConsiderList, const float ServerTickTime)
+{
+	const bool bSpatialNetworking = GetDefault<UGeneralProjectSettings>()->UsesSpatialNetworking();
+
+	if (!bSpatialNetworking || AuthorityDebugger == nullptr)
```

Comment:
In the unlikely event that bSpatialNetworking was false but AuthorityDebugger was true, the results would be wrong?

Maybe better to only check the first, but lower down assert on the second?

Reply:
To be honest I think the inclusion or exclusion of non-auth actors from the Super version of this function, based on the settings boolean, combined with if/else code below, might be hard to new arrivals to this code to read?

Another option would be to change the API of this function to include a user definable predicate function for excluding actors... in spatial this could exclude non-auth actors, whilst also verifying for unauthorised data changes.